kore/app/repl/Main.hs

```diff
@@ -215,21 +226,32 @@ mainWithOptions
                 exitFailure
             else
                 lift
-                $ SMT.runSMT smtConfig (swappableLogger mLogger)
-                   $ do
-                        give
-                            (MetadataTools.build indexedModule)
-                            (declareSMTLemmas indexedModule)
-                        proveWithRepl
-                            indexedModule
-                            specDefIndexedModule
-                            Nothing
-                            mLogger
-                            replScript
-                            replMode
-                            outputFile
+                $ withLogger koreLogOptions
```

Comment:
Where did `SMT.runSMT` go?

Reply:
I used it inside `logActionToIO` at line 253

Reply:
Could we rename that function to `runSMT`, or something like that? It's not clear from the name that it has anything to do with SMT.